twill-core/src/main/java/org/apache/twill/internal/kafka/client/SimpleKafkaConsumer.java

```diff
@@ -68,7 +69,7 @@
 /**
  * A {@link KafkaConsumer} implementation using the scala kafka api.
  */
-final class SimpleKafkaConsumer implements KafkaConsumer {
+public final class SimpleKafkaConsumer implements KafkaConsumer {
```

Comment:
Right, this can be done in the unit test. To phrase my question better, what can be a better way to perform the same logic as `getLastOffset` outside of `SimpleKafkaConsumer`? Maybe expose the parameter passed to `SimpleKafkaConsumer` constructor in `ZKKafkaClientService`?